charts/super-agent/charts/super-agent-deployment/tests/deployment_superagent_securityContext_test.yaml

```diff
@@ -3,6 +3,9 @@ templates:
   - templates/deployment-superagent.yaml
   - templates/configmap-superagent-config.yaml
   - templates/configmap-subagent-configs.yaml
+release:
```

Comment:
why this change is needed?

Reply:
leftover, since i ended up moving the test to other file